tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "strictPropertyInitialization": false,
     "declaration": true,
     "baseUrl": ".",
-    "outDir": "./build"
+    "outDir": "./build",
+    "skipLibCheck": true
```

Comment:
Using TypeScript 4.4.x without this setting there were some excessive type errors being thrown out from `node_modules`

Reply:
I don't think that this is a good idea. what's the typescript complain?

Reply:
```shell
$ tsc --noEmit
node_modules/@types/express-serve-static-core/index.d.ts:30:10 - error TS2305: Module '"qs"' has no exported member 'ParsedQs'.

30 import { ParsedQs } from 'qs';
            ~~~~~~~~

node_modules/@types/express/index.d.ts:58:29 - error TS2694: Namespace 'serveStatic' has no exported member 'RequestHandlerConstructor'.

58     var static: serveStatic.RequestHandlerConstructor<Response>;
                               ~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@types/serve-static/node_modules/@types/express-serve-static-core/index.d.ts:501:18 - error TS2430: Interface 'Response<ResBody>' incorrectly extends interface 'ServerResponse'.
  Types of property 'req' are incompatible.
    Type 'Request<ParamsDictionary, any, any, Query> | undefined' is not assignable to type 'IncomingMessage'.
      Type 'undefined' is not assignable to type 'IncomingMessage'.

501 export interface Response<ResBody = any> extends http.ServerResponse, Express.Response {
                     ~~~~~~~~


Found 3 errors.
```

these appear to be transitive dependencies of the `apollo-server-express` dependency